lib/streamlit/elements/__init__.py

```diff
@@ -68,9 +69,6 @@
     "table",
     "text",
     "vega_lite_chart",
-    "write",
-]
-FILESYSTEM_ELEMENTS = [
-    "audio",
     "video",
+    "write",
```

Comment:
Is there a reason why write is going to filesystem_elements?

Reply:
It isn't. We're getting rid of that list entirely, the diff is just a little confusing because [delete these lines and move "write" down] is a smaller change than [delete the whole FS elements list and move "write" up] to the diff algorithm